onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -900,7 +900,13 @@ Status Expand_8<T>::Compute(OpKernelContext* context) const {
 
   // Turn the shape tensor data into an actual shape
   const auto* p_shape = tensor_shape.template Data<int64_t>();
-  std::vector<int64_t> shape{p_shape, p_shape + tensor_shape.Shape().Size()};
+  std::vector<int64_t> shape(tensor_shape.Shape().Size());
+  for (auto i = 0; i < tensor_shape.Shape().Size(); i++) {
+    // According to the ONNX spec: "Dimensions are right alignment;
```

Comment:
is this the Expand change you missed to revert?

Reply:
You are right! Reverted it now.